VirtoCommerce.CatalogModule.Data/Services/CategoryMover.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using System.Linq;
+
+using VirtoCommerce.CatalogModule.Web.Model;
+using VirtoCommerce.CatalogModule.Web.Services;
+using VirtoCommerce.Domain.Catalog.Services;
+using VirtoCommerce.Platform.Core.Common;
+
+using VC = VirtoCommerce.Domain.Catalog.Model;
```

Comment:
Not clear what means VC. "domain" or "coreModel" would be self-explanatory, and will be coherent to the existing naming style.